src/redux/epics/attach-access-type.js

```diff
@@ -0,0 +1,30 @@
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/of';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/catch';
+
+import {
+  ATTACH_ACCESS_TYPE,
+  attachAccessTypeSuccess,
+  attachAccessTypeFailure,
+  addAccessType,
+} from '../actions';
+
+export default ({ accessTypesApi }) => (action$, store) => {
+  const { getState } = store;
+  const state = getState();
+
+  return action$
+    .filter(action => action.type === ATTACH_ACCESS_TYPE)
+    .mergeMap(action => {
+      const { payload: accessType } = action;
+
```

Comment:
Current variable could be destructuring  in an arguments above `({ payload: accessType }) => {...}`
And the same can be done in simular cases below, just FYI